src/components/QRCodeReader.js

```diff
@@ -62,8 +62,8 @@ export default ({
     setIsFocusedScreen(true);
 
     return () => {
-      focusEvent.remove();
-      blurEvent.remove();
+      focusEvent();
+      blurEvent();
```

Comment:
React Navigation v5.x event listeners do not have the `remove` method, and can be removed by calling them directly. ( See [API Docs](https://reactnavigation.org/docs/5.x/navigation-events#navigationaddlistener) )